package.json

```diff
@@ -11,17 +11,19 @@
   "bugs": "https://github.com/wayfair/tungstenjs/issues",
   "main": "tungsten.js",
   "scripts": {
+    "clean": "rm dist/*.js",
```

Comment:
Leave it for now (someone might be using it) and we can create a more compatible version later.